modules/ROOT/pages/deploy-to-rtf.adoc

```diff
@@ -1,33 +1,24 @@
 = Deploy Applications to Runtime Fabric Using the Mule Maven Plugin
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
 
+<<<<<<< HEAD
 include::mule-runtime::partial$mmp-concept.adoc[tag=deprecatedVersionsWarning]
+=======
+include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=intro]
+>>>>>>> v4.3
```

Comment:
TODO/MERGE CONFLICT: keep/incorporate changes from v4.3?

Reply:
@Cristian-Venticinque

Reply:
Please add changes from v4.3 to v4.4, for v4.3 I turned this deploy-to-rtf article into a partial, because the same file is reused in RTF docs. The current version of this article is just a page title and include lines, no content.